go/src/meguca/server/router.go

```diff
@@ -238,3 +239,11 @@ func crossRedirect(w http.ResponseWriter, r *http.Request) {
 		text500(w, r, err)
 	}
 }
+
+// Health check to ensure server is still online
+func healthCheckTXT(w http.ResponseWriter, r *http.Request) {
+	var buf bytes.Buffer
```

Comment:
Okay. Lesson time. See https://golang.org/pkg/net/http/#ResponseWriter. It has the Write() method. So you can just use that to write to the HTTP response. As for casting string to []byte: `[]byte("foo")`.